framework/include/cppmicroservices/Any.h

```diff
@@ -207,7 +208,7 @@ class US_Framework_EXPORT Any
    * \param other The Any to copy
    */
   Any(const Any& other)
```

Comment:
All code changes in this submission were made with clang-tidy. Can we make further changes like those described in the future?

Reply:
In my experience, the result of automated tools like this almost always need some tweaking. At leas to make the code base consistent. The inconsistencies are small and I am just concerned that they will stay if not worked on in this PR.

If you say it will not be like that, fair enough :) Then let's do them in a subsequent change set.

Reply:
Actually, I took a look at this file, and it appears to me as if both the move constructor and move assignment operator are implemented. The move constructor is around line 219 and looks like this:

  Any(Any&& other)
    : _content(std::move(other._content))
  {}

And the move assignment operator is at line 302 and looks like this:

  Any& operator=(Any&& rhs)
  {
    _content = std::move(rhs._content);
    return *this;
  }

Reply:
Great, I must have overlooked this in the diff view when looking at the PR changes.